src/NuGet.Core/NuGet.Build.Tasks/ErrorForMismatchRestoreOutputPathTask.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using NuGet.Commands;
+
+namespace NuGet.Build.Tasks
+{
+    public class ErrorForMismatchRestoreOutputPathTask : Task
```

Comment:
As we discussed in chat, can you please remove this error message, because it will break the advanced scenarios (namely dotnetclitool and global tools).